test/unittest/testAlignment.py

```diff
@@ -8,136 +8,124 @@
 sepp._DEBUG = True
 from sepp.alignment import MutableAlignment, ReadonlySubalignment,\
     ExtendedAlignment
-from sepp.problem import SeppProblem        
+from sepp.problem import SeppProblem
```

Comment:
my editor "Atom" automatically fixed some code style issues for the files I modified; it's basically whitespaces